packages/office-ui-fabric-react/src/utilities/test/TestUtilities.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactWrapper } from 'enzyme';
+import * as sinon from 'sinon';
+/**
+ * Injects a function call prior to running componentDidUpdate for a component
+ * rendered using enzyme deep rendering.
+ * @param wrapper The enzyme deep rendering wrapper whose onComponentDidUpdate
+ * @param fn The function to run prior to onComponentDidUpdate
+ */
+export function runPriorToComponentDidUpdate(wrapper: ReactWrapper<any, any>, fn: () => void) {
+  // tslint:disable-next-line:no-string-literal
+  const originalComponentDidUpdate = wrapper.instance()['componentDidUpdate'];
+  // tslint:disable-next-line:no-string-literal
+  wrapper.instance()['componentDidUpdate'] = function (prevProps: any) {
+    fn();
+    originalComponentDidUpdate.call(this, prevProps);
+  };
+}
+
+/**
+ * Replaces the enzyme render function with a spy that can be used to count how many
+ * times the render function was called in a test.
+ * @param wrapper The wrapper around the component instance for which you want to spy upon
+ */
+export function getRenderSpy(wrapper: ReactWrapper<any, any>): sinon.SinonSpy {
```

Comment:
get doesn't imply the wrapper render function is destroyed and replaced with a spy. Can this be renamed to setRenderSpy?